tests/test_structs.py

```diff
@@ -52,6 +52,7 @@
     (r"\2plus\2is\4", r"\2plus\2is\4"),
     (r"foo\bar", r"foo\bar"),
     (r"foo\41r", r"foo\41r"),
+    (r"call\20me", r"call\5c20me"),
```

Comment:
@lovetox This doesn’t quite fit into the `exceptions` section, maybe put it into a new "additional test vectors" section.

This is also a good candidate for the XEP.